lib/ts/version.ts

```diff
@@ -12,6 +12,6 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-export const package_version = "17.0.5";
+export const package_version = "17.0.6";
```

Comment:
This should be a minor version increase (it's a new feature, but it's backwards compatible, not a fix).